beamer/agent/data/agent.conf.example

```diff
@@ -2,6 +2,7 @@ log-level = "debug"
 deployment-dir = "deployments/goerli"
 fill-wait-time = 120
 unsafe-fill-time = 600
+confirmation-blocks = 0
```

Comment:
What is the meaning of the confirmation-blocks on the top level?

Reply:
is it meant to be a default?

Reply:
yes it will be default. the first suggestion was using the structure like we are using for poll_period and chain based poll_period so i have added also this one as default.

Reply:
Just to clarify: 
The agent has the option to configure the global default, plus for chain specific values, which override the default? 
Additionally, we have an internal default which is used if no global default is set? 
And this counts both for poll period and confirmation blocks?

Reply:
(if true, sounds to me like an overkill)

Reply:
> Just to clarify: The agent has the option to configure the global default, plus for chain specific values, which override the default? Additionally, we have an internal default which is used if no global default is set? And this counts both for poll period and confirmation blocks?

Yes, exactly.

I do admit it may seem as a bit of an overkill and most likely not needed (at least not right now), but OTOH, having it in this manner is more in line with how people generally expect configuration to behave, so I see it not as "we're definitely not going to need this" but rather "people expect config files to work a certain way, so let's make it intuitive for them, even though they may not have an immediate need for it".